plugin/geoip/testdata/create_databases.go

```diff
@@ -0,0 +1,117 @@
+// This package is a helper function to create the mmdb databases used for testing in the
+// geoip plugin.
+package main
```

Comment:
I don't think we need this binary, just check in a test mmdb, maybe put in some pointers on how to create one yourself.

Reply:
For the test cases to work is important to respect the schema and the values set in them, I also think this is the best way for future readers to discover and understand the mmdb file structure and current values.

Keeping the exact schema and values around is also very handy in case a simple modification like adding one extra value, or changing the value of a field is needed.

If you push for it I don't mind doing what you suggest, but I feel the value of keeping that code around and not just a few hints on how you could create one database, is higher than removing it.

If it really bothers you to have a main file there, would it work for you to just move that to a `README.md` in the testdata directory?